bu/Processor.cpp

```diff
@@ -450,21 +450,22 @@ void Processor::writeEptFile(const std::string& filename, pdal::PointTableRef ta
     pdal::Options wopts;
     wopts.add("extra_dims", "all");
     wopts.add("software_id", "Entwine 1.0");
-    wopts.add("compression", "lazperf");
+    wopts.add("compression", "laszip");
     wopts.add("filename", filename);
     wopts.add("offset_x", m_b.offset[0]);
     wopts.add("offset_y", m_b.offset[1]);
     wopts.add("offset_z", m_b.offset[2]);
     wopts.add("scale_x", m_b.scale[0]);
     wopts.add("scale_y", m_b.scale[1]);
     wopts.add("scale_z", m_b.scale[2]);
+    wopts.add("minor_version", 4);
```

Comment:
I thought QGIS expected LAZ 1.2 all the time?

Reply:
Then we can't fix this problem. Do you want to ask the powers that be?

Reply:
Well, @wonder-sk is the power-that-be here that matters 😄

Reply:
Ah it took me a bit to realize that this PR is meant to fix #75 ... right now QGIS expects LAZ 1.2 because we use laz-perf 2.x for parsing. But if this is the correct fix for the issues, that's fine, as we'll need to switch to laz-perf 3.0 soon anyway to support COPC (and laz-perf 3.0 reads LAZ 1.4 if I got it right?)

Reply:
Lazperf 2 support LAZ 1.4 as well.

Reply:
@wonder-sk : We're moving PDAL to requiring C++17 to build. Do you have any issues with the same for Untwine?

Reply:
That's fine - it looks like in QGIS we have switched to C++17 last year - https://github.com/qgis/QGIS/commit/a54efae8923ee0d3bd7ab9b5201a65342a7eafe1